micro_mg3_0.F90

```diff
@@ -2267,20 +2316,32 @@ subroutine micro_mg_tend ( &
         ! estimates are only approximate since other process terms haven't been limited
         ! for conservation yet
 
-        ! first limit ice deposition/nucleation vap_dep + mnuccd
-        dum1 = vap_dep(i,k) + mnuccd(i,k)
+        ! first limit ice deposition/nucleation vap_dep + mnuccd + vap_deps
+        dum1 = vap_dep(i,k) + mnuccd(i,k) + vap_deps(i,k)
         if (dum1 > 1.e-20_r8) then
-           dum = (q(i,k)-qvi(i,k))/(1._r8 + xxls_squared*qvi(i,k)/(cpp*rv*t(i,k)**2))*rdeltat
+           dum = (q(i,k)-qvi(i,k))/(1._r8 + xxls_squared*qvi(i,k)/(cpp*rv*t(i,k)**2))/deltat
```

Comment:
Better to use `*rdeltat` instead of `/deltat`.

Reply:
Done.